packages/backend/src/services/GitIntegrationService/GitIntegrationService.ts

```diff
@@ -245,107 +249,126 @@ Affected charts:
         };
     }
 
+    static generateDiff(original: string, updated: string): string {
```

Comment:
we might want to return this diff also on a  `previewPr` endpoint, before the PR is created, to show the user the changes in the UI

Reply:
Don't we need the info about `added/removed` for the UI to apply the green/red colour? what is the downside of sending the `diff` array?